packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -65,15 +65,15 @@ from audioseal import AudioSeal
 model = AudioSeal.load_generator("${model.id}")
 # pass a tensor (tensor_wav) of shape (batch, channels, samples) and a sample rate
 wav, sr = tensor_wav, 16000
-	
+
```

Comment:
Personally, not a fan of unrelated formatting changes, but I'll let @Wauplin or @hanouticelina comment on best practices in this regard.

Reply:
Thanks for the review. Happy to revert these changes if you feel it is needed.